release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -480,6 +480,13 @@ For more information, see xref:../edge_computing/ztp-deploying-far-edge-sites.ad
 [id="ocp-4-16-insights-operator_{context}"]
 === Insights Operator
 
+[id="ocp-4-16-etcd-certificates_{context}"]
+=== Security
+
+Currently, `etcd` certificates are set to expire in two years. A new signer certificate authority (CA), `openshift-etcd` is now available to sign certificates. It is contained in a trust bundle with the existing CA. Two CA secrets are also available - `etcd-signer` and `etcd-metric-signer`, which are available for rotation.
```

Comment:
`Currently, `etcd` certificates are set to expire in two years.`

that I would leave out, this doesn't change and has been the case since a long time